crates/sui-core/src/authority/shared_object_version_manager.rs

```diff
@@ -469,6 +487,13 @@ mod tests {
                 ],
                 9,
             ),
+            generate_shared_objs_tx_with_gas_version(
+                &[
+                    (id1, init_shared_version_1, true),
```

Comment:
Consider changing one of these ids to SUI_RANDOMNESS_STATE_OBJECT_ID to check that it can be assigned to SequenceNumber::RANDOMNESS_UNAVAILABLE

Reply:
thanks, I had meant to do that and forgot! done